pkg/kubelet/volume/host_path/host_path.go

```diff
@@ -54,6 +54,12 @@ func (plugin *hostPathPlugin) CanSupport(spec *api.Volume) bool {
 	return false
 }
 
+func (plugin *hostPathPlugin) GetAccessModes() []api.AccessModeType {
+	return []api.AccessModeType{
+		api.ReadWriteOnce,
```

Comment:
Technically, hostPath and gcePD are both "read-write-many-if-same-node" but we probably don't want to represent that.

but OK